### PR TITLE
[dotnet] Remove System.Drawing.Common as package dependency

### PR DESCRIPTION
**Pull request by nvborisenko**
This is continuation of https://github.com/SeleniumHQ/selenium/pull/12777, please review that first.

### Description
I think selenium is not responsible to convert images from one format to another. It is already enough for users to deal with images in `png` format, even it is possible to save image as file.

If user wants to convert image, then he is able to get image as bytes and use any other 3rd-party library especially designed to work with graphics.

### Motivation and Context
Continue my way to reduce package dependencies.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by titusfortner** [comment]
I think where this is coming from is that Java supports:
* Base64
* Bytes
* File (stored as png)

If .NET doesn't allow saving as png in default libraries...

We can't just remove this. We can deprecate the method.
We should have a different method to return it as a Stream so a user knows they need to do something else with it.

Another option that might be nice is to move the ability to save a stream as a PNG to the support package.

**Comment by nvborisenko** [comment]
Now all users will get a warning. And those, who are really converting image to different formats, will get exception at runtime.

**Comment by titusfortner** [comment]
Please rebase on trunk and fix conflicts so I can better see what this is doing. :)

**Comment by titusfortner** [comment]
> those, who are really converting image to different formats, will get exception

I think we want to do this in stages. If we are removing functionality we should mark it as obsolete for a couple releases, then remove it.

**Comment by nvborisenko** [comment]
Agree, I will prepare separate PR just to mark methods as obsolete. Keeping this one as open until we decide completely remove it.

File: dotnet/src/webdriver/Screenshot.cs
Comment (nvborisenko) [inline dotnet/src/webdriver/Screenshot.cs:1]: As a reminder for myself: we need adjust tests accordingly.

Comment (nvborisenko) [reply dotnet/src/webdriver/Screenshot.cs:1]: Done.

**Comment by nvborisenko** [comment]
I think we are ready to land it.

**Comment by ckuetbach** [comment]
Two weeks for a breaking change in a minor release is somewhat annoying.

The Obsolete message was merged 22nd October and the breaking change was introduced Nov 10. (https://github.com/SeleniumHQ/selenium/pull/12878)

**Comment by nvborisenko** [comment]
The obsolete message shipped with v4.14 (25 Sep) and breaking change shipped with v4.16 (06 Dec). Selenium project doesn't follow semantic versioning, sorry for inconvenience.

**Pull request merged**